locales/en.json

```
@@ -552,6 +552,8 @@
       "use_star_favorite_icon": "Use star favorite icon",
       "user_picker": "User Picker",
       "user_picker_description": "Displays all avatars of logged accounts in the bottom-left so you can switch quickly between them.",
+      "view_transitions": "View Transitions",
+      "view_transitions_description": "Navigate pages while transitioning the common elements between them.",
```

This should be more user oriented, it's very technical, something like  `😍 Enable beutiful animations  when navigating between pages` (the emoji shouldn't be included) would be better

---

/cc @patak-dev any suggestion here?

---

I think maybe we should just say:
```suggestion
      "view_transitions_description": "Animate transitions.",
```